bedrock/settings/static_media.py

```diff
@@ -396,6 +396,21 @@
         ),
         'output_filename': 'css/firefox_firstrun-bundle.css',
     },
+    'firefox_firstrun_quantum': {
+        'source_filenames': (
+            'css/base/notification-banner.less',
+            'css/firefox/firstrun/firstrun_quantum.less',
+            'css/base/mozilla-fxa-iframe.less',
+        ),
+        'output_filename': 'css/firefox_firstrun_quantum-bundle.css',
+    },
+    'firefox_firstrun_membership': {
```

Comment:
This was removed in https://github.com/mozilla/bedrock/pull/5243. Rebasing from master might clean this up automatically but if not we should remove it here.